src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,137 @@
+p,
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px 0px 0px 0px;
+  border: 1px solid #f3f3f3;
+  opacity: 0px;
+
+  padding: 32px 16px 16px;
+  margin: 0;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+    // justify-content: center;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+
+    color: #060b35;
+  }
+
+  &__code {
+    display: block;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+
+    color: #616070;
+
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    margin-bottom: 24px;
+
+    &-amount {
+      display: block;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+
+      color: #060b35;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 16px;
```

Comment:
The bottom margin for `.card__price` is set to `16px`. Verify that this margin is consistent with the design and consider using a SCSS variable for consistent spacing.